grc/CMakeLists.txt

```diff
@@ -18,4 +18,4 @@
 # Boston, MA 02110-1301, USA.
 
 file(GLOB yml_files "*.yml")
-install(FILES ${yml_files} DESTINATION ${GRC_BLOCKS_DIR})
+install(FILES ${yml_files} DESTINATION share/gnuradio/grc/blocks)
```

Comment:
would it be a better idea to add this value as a default to the `GRC_BLOCKS_DIR` variable?

Reply:
Sure. I have made a new commit 173e7cf58e52b8073482880fc3443da88b2956e8 that does this.